qiskit/transpiler/passes/calibration/creators.py

```diff
@@ -0,0 +1,472 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2021.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Calibration creators."""
+
+from abc import abstractmethod
+from typing import List, Union
+
+import math
+import numpy as np
+
+from qiskit.circuit import Instruction as CircuitInst
+from qiskit.circuit.library.standard_gates import RZXGate
+from qiskit.dagcircuit import DAGCircuit
+from qiskit.exceptions import QiskitError
+from qiskit.providers import basebackend
+from qiskit.pulse import (
+    Play,
+    Delay,
+    ShiftPhase,
+    Schedule,
+    ScheduleBlock,
+    ControlChannel,
+    DriveChannel,
+    GaussianSquare,
+)
+from qiskit.pulse.instruction_schedule_map import InstructionScheduleMap, CalibrationPublisher
+from qiskit.pulse.instructions.instruction import Instruction as PulseInst
+from qiskit.transpiler.basepasses import TransformationPass
+
+
+class CalibrationCreator(TransformationPass):
```

Comment:
```suggestion
class CalibrationBuilder(TransformationPass):
```

This is more standard within compiler terminology.

Reply:
Code is upgraded in https://github.com/Qiskit/qiskit-terra/pull/6759/commits/e16f931a672ea55ef7804a76ace1f6362c18283d